packages/core/.storybook/preview.js

```diff
@@ -42,6 +42,26 @@ const decorators = [
 const parameters = {
   controls: { expanded: true, hideNoControlsWarning: true },
   layout: 'centered',
+  options: {
+    storySort: (a, b) => {
+      if (a[1].kind === b[1].kind) {
+        return 0;
+      } else {
+        if (
+          a[1].kind.startsWith('Cloud & Cognitive/Released/') &&
+          b[1].kind.startsWith('Cloud & Cognitive/Canary/')
+        )
+          return -1;
+        if (
+          b[1].kind.startsWith('Cloud & Cognitive/Released/') &&
+          a[1].kind.startsWith('Cloud & Cognitive/Canary/')
+        )
+          return 1;
+        console.log(a[1].kind, b[1].kind);
+        return a[1].id.localeCompare(b[1].id, undefined, { numeric: true });
+      }
+    },
```

Comment:
Would it make more sense to leverage the [`order` property](https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#sorting-stories) on `storySort` here? I feel it would be both easier on the eyes and to maintain

Reply:
Tried this, but it did not work. Only seems to work on first level meaning canary would be above released.

```
  options: {
    storySort: {
      order: ['Cloud & Cognitive/Released', 'Cloud & Cognitive/Canary', 'Legacy']
    },
  },
```